--- Gtk2-GLExt-0.90/Makefile.PL.orig	2004-10-29 02:18:43.000000000 +0200
+++ Gtk2-GLExt-0.90/Makefile.PL	2007-06-02 18:41:16.328630606 +0200
@@ -42,14 +42,14 @@
 %pkgcfg = ExtUtils::PkgConfig->find ('gtkglext-1.0 >= '
 				     .$build_reqs{GtkGLExt});
 
+mkdir 'build', 0777;
+
 our @xs_files = <xs/*.xs>;
 our %pod_files = (
 	'GLExt.pm'          => '$(INST_MAN3DIR)/Gtk2::GLExt.$(MAN3EXT)',
 	Glib::MakeHelper->do_pod_files (@xs_files),
 );
 
-mkdir 'build', 0777;
-
 #
 # autogeneration
 #
